lms/djangoapps/instructor/services.py

```diff
@@ -85,4 +86,4 @@ def is_course_staff(self, user, course_id):
         Returns True if the user is the course staff
         else Returns False
         """
-        return auth.user_has_role(user, CourseStaffRole(CourseKey.from_string(course_id)))
+        return auth.user_has_role(user, CourseStaffRole(course_key_from_string_or_404(course_id)))
```

Comment:
is this an appropriate place to use this helper? Is it only being called by a view?